src/runtime/pythonengine.cs

```diff
@@ -582,28 +573,8 @@ public static void Exec(string code, IntPtr? globals = null, IntPtr? locals = nu
         /// <returns>The native thread ID.</returns>
         public static ulong GetNativeThreadID()
```

Comment:
If we keep this, it should be renamed to `GetPythonThreadId` and doesn't this require the GIL in general?

Reply:
Can you please explain why we need GIL here and not in the Interrupt method? GIL is used for both in the tests already.

Reply:
And renaming would still be nice, sorry for the back and forth, I misread the docs.